README.md

```diff
@@ -48,6 +48,7 @@ Prometheus-kafka-adapter listens for metrics coming from Prometheus and sends th
 - `BASIC_AUTH_PASSWORD`: basic auth password to be used for receive endpoint, defaults is no basic auth.
 - `LOG_LEVEL`: defines log level for [`logrus`](https://github.com/sirupsen/logrus), can be `debug`, `info`, `warn`, `error`, `fatal` or `panic`, defaults to `info`.
 - `GIN_MODE`: manage [gin](https://github.com/gin-gonic/gin) debug logging, can be `debug` or `release`.
+- `MATCH`: filter which metrics and labels to send to kafka. Used to limit the amount of data that is sent to kafka - by default everything is send. Example:   "['metric-1{label-1=\"label-value-1\"}','metric-2']"
```

Comment:
So this is intended to be an inclusive filter (i.e. only things in the filter list will be forwarded)? Can we add a note about the _type_ of filter the user will experience?

Reply:
Yes, sure - as it is in the moment in the example:  everything that is <metric-1> with <label-1> with value of the label <label-value-1> and everything from <metric-2> will be sent to kafka, everything else will be dropped. Do you want me to add the PR - will push to it if required.

Reply:
I've merged it because this description is much better than nothing. Any ideas to improve its clarity are welcome.